### PR TITLE
[OSAC-284] Update dependent packages (platform, Studio, Api) to match other packages (21.12-preview)

### PR DESCRIPTION
**Pull request by mateisimionescu**
Description
Update dependent packages
How to test
Open Activity package in Studio Web
Jira
OSAC-284
Label:
24S

### CODE REVIEW

File: Activities/Directory.build.targets [hunk 1]
```diff
@@ -12,9 +12,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Update="UiPath.Robot.Api" Version="21.6.0-beta.4360" PrivateAssets="All" />
```
Comment (alcxs) [inline Activities/Directory.build.targets]: please check versions for packages that are 21.6.*

File: Activities/Directory.build.targets [hunk 2]
```diff
@@ -12,9 +12,10 @@
...
-	  <PackageReference Update="UiPath.Activities.Api" Version="21.6.0-beta.4360" PrivateAssets="All" />
+    <PackageReference Update="UiPath.Activities.Api" Version="21.6.0-beta.4360" PrivateAssets="All" />
     <PackageReference Update="UiPath.Activities.Contracts" Version="21.6.0-beta.4360" PrivateAssets="All" />
```
Comment (alcxs) [inline Activities/Directory.build.targets]: same here

**Review by alcxs — CHANGES_REQUESTED** [review]
please check versions for ALL packages and update accordingly

**Review by alcxs — APPROVED** [review]

File: Activities/Directory.build.targets [hunk 3]
```diff
@@ -1,19 +1,19 @@
 <Project>
   <ItemGroup Condition="$(TargetFramework.StartsWith('net5'))"> 
-    <PackageReference Update="System.Activities" Version="1.0.0-20201117-04" />
-    <PackageReference Update="UiPath.Workflow" Version="1.0.0-20201117-04" />
-    <PackageReference Update="System.Activities.Core.Presentation" Version="1.0.0-alpha004" />
-    <PackageReference Update="System.Activities.ViewModels" Version="1.0.0-20211101.1"/>
+    <PackageReference Update="System.Activities" Version="5.0.0-20210615-01" />
+    <PackageReference Update="UiPath.Workflow" Version="5.0.0-20210615-01" />
+    <PackageReference Update="System.Activities.Core.Presentation" Version="5.0.0-20210724.1" />
+    <PackageReference Update="System.Activities.ViewModels" Version="1.0.0-20211215.1"/>
```
Comment (ovidiuiliescu) [inline Activities/Directory.build.targets]: At least for ViewModels there's a newer version out, please use that.

**Review by ovidiuiliescu — CHANGES_REQUESTED** [review]

**Review by ovidiuiliescu — APPROVED** [review]

**Pull request merged**